src/FactCheck.jl

```diff
@@ -98,34 +98,34 @@ function Base.show(io::IO, f::Failure)
             # Fancy helper fact
             fcFunc = _factcheck_function(args[2])
             if haskey(FACTCHECK_FUN_NAMES, fcFunc)
-                println(io, replace_lf(string("\tExpected: ", sprint(show, f.lhs), " ", FACTCHECK_FUN_NAMES[fcFunc], " ", sprint(show, f.rhs))))
+                print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.lhs), " ", FACTCHECK_FUN_NAMES[fcFunc], " ", sprint(show, f.rhs))))
             else
-                println(io, replace_lf(string("\tExpected: ", sprint(show, f.lhs), " --> ", fcFunc, "(", sprint(show, f.rhs), ")")))
+                print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.lhs), " --> ", fcFunc, "(", sprint(show, f.rhs), ")")))
             end
         else
             # Normal equality-test-style fact
-            println(io, replace_lf(string("\tExpected: ", sprint(show, f.rhs), "\n", "\tOccurred: ", sprint(show, f.lhs))))
+            print(io, replace_lf(string("<:LF:>  Expected: ", sprint(show, f.rhs), "<:LF:>  Occurred: ", sprint(show, f.lhs))))
         end
     else
         error("Unknown fact type: ", f.fact_type)
     end
-    println(io, "\n<COMPLETEDIN::>")
+    println(io, "\n\n<COMPLETEDIN::>")
 end
 
 function Base.show(io::IO, e::Error)
-    println(io, "\n<IT::>", e.meta.msg != nothing ? "$(e.meta.msg)" : format_fact(e.expr))
-    println(io, "\n<ERROR::>Test Errored")
-    println(io, replace_lf(sprint(showerror, e.err, e.backtrace)))
+    println(io, "\n<IT::>", replace_lf(e.meta.msg != nothing ? "$(e.meta.msg)" : format_fact(e.expr)))
+    println(io, "\n<ERROR::>", replace_lf(sprint(showerror, e.err)))
+    println(io, "\n<LOG::-Stack trace>", replace_lf(sprint(showerror, e.err, e.backtrace)))
     println(io, "\n<COMPLETEDIN::>")
 end
 
 function Base.show(io::IO, s::Success)
     if s.rhs == :fact_throws_error
-        println(io, "\n<IT::>", s.meta.msg != nothing ? "$(s.meta.msg)" : "Throws Error")
+        println(io, "\n<IT::>", replace_lf(s.meta.msg != nothing ? "$(s.meta.msg)" : "Throws Error"))
         println(io, "\n<PASSED::>Test Passed")
         println(io, "\n<COMPLETEDIN::>")
     else
-        println(io, "\n<IT::>", s.meta.msg != nothing ? "$(s.meta.msg)" : format_fact(s.expr))
+        println(io, "\n<IT::>", replace_lf(s.meta.msg != nothing ? "$(s.meta.msg)" : format_fact(s.expr)))
```

Comment:
Just noticed these are inconsistent with `Base.show(io::IO, f::Failure)`. The same tests will have different `<IT::>` depending on the result.

Reply:
I think `meta.msg` hack is there because there was no other way to set a descriptive test case name.

Reply:
I'll take a look and see what can be done. The `it` being a synthetic thing does not help tho. As far as I gee it, you want the describes and its be the same for passing and failing tests?

Reply:
Yeah, currently, the failed tests are using `f.expr` when `f.meta.msg` exists. If `f.expr` is good enough, then it should be changed to use it everywhere. If not, then it should be reverted.

Reply:
Please see #3 .

I decided to go with just  `f.expr` in titles of `<IT::>` because:
- IMO it looks good enough at worst, and quote good at best, because a stringized expression describes a test case quite well if you ask me
- I think that using what-should-be an assertion message in place of a test title is a CW-specific hack, and if users dont know this, their tests end up with an awkward titles. Just like my initial attempts which displayed `> Incorrect answer for n=10` as a title of a passing test. It can  stay this way if you prefer (which I would not like), but then it would have to be documented somewhere. I think that with the proposed solution, there is at least a clear distinction what is a title of DESCRIBE, what is a title of IT, and what is a custom message. If there would be some hack/workaround for custom titles of ITs, i'd propose a CW-specific assertion function rather than ~~misusing~~ reusing elements of  an existing API and changing their established meaning.